### ISSUE TITLE
YOLOv3 cfg file

### ISSUE THREAD

**Issue by canyilmaz90** [WongKinYiu/PartialResidualNetworks#5]
Hello, firstly thanks for sharing this great work. I want to test this for yolov3 (not tiny), can you share config file for it?
Thanks and regards

**Comment by WongKinYiu** [comment]
@canyilmaz90 Hello,

Do you mean "Darknet53-PRN with YOLOv3 head" or "Darknet53 with YOLOv3-PRN head"?

**Comment by canyilmaz90** [comment]
@WongKinYiu Hi,

I mean the one you named YOLO-v3-PRN in your paper. I think it's "Darknet53-PRN with YOLOv3 head", but I'm not sure. Btw, what is the difference between two? And is there a possibility of "Darknet53-PRN with YOLOv3-PRN head"?

**Comment by WongKinYiu** [comment]
Yes, it's "Darknet53-PRN with YOLOv3 head".
I did not test "Darknet53-PRN with YOLOv3-PRN head".

I will share the cfg files in few days.
I am busy in AVSS 2019 & ICIP 2019 now.

**Comment by canyilmaz90** [comment]
Thanks a lot! Good luck with the conferences :)

**Comment by WongKinYiu** [comment]
cfg file for YOLO-v3-PRN in the paper
[darknet53-prn-yolov3.cfg.txt](https://github.com/WongKinYiu/PartialResidualNetworks/files/3629050/darknet53-prn-yolov3.cfg.txt)

**Comment by canyilmaz90** [comment]
Thank you so much. After I train with these configurations, I will give you feedback.

**Comment by WongKinYiu** [comment]
The weight file of YOLO-v3-PRN pre-trained model is too big, i can not upload it to github.

**Comment by canyilmaz90** [comment]
Thanks anyway. Maybe you could put it in a cloud account, such as google drive

**Comment by LukeAI** [comment]
@WongKinYiu Would you also consider releasing Yolo-v3_FPRN ?

**Comment by WongKinYiu** [comment]
@LukeAI Hello,

For YOLO-v3-FPRN, I modified shortcut_cpu (blas.c) and shortcut_gpu (blas_kernels.cu) instead of modified the cfg file.

for example, change `int minc = (c1 < c2) ? c1 : c2;` in shortcut_cpu to `int minc = (c1 < c2) ? c1//2 : c2//2;`. then the cfg file is totally same as that for YOLO-v3.

**Comment by iraadit** [comment]
Hi @WongKinYiu 

Will you share the YOLO-v3-PRN pre-trained COCO weights? On Google Drive, Dropbox or similar sharing service?

Thank you

**Comment by WongKinYiu** [comment]
@iraadit hello,

we will release our new model in few days.
i ll post the link here.

**Comment by WongKinYiu** [comment]
@iraadit 

we release our new model at https://github.com/WongKinYiu/CrossStagePartialNetworks

**Comment by LukeAI** [comment]
I'd also like to request any pretrained weights you might have for YOLO-v3-PRN

**Comment by reactivetype** [comment]
> Do you mean "Darknet53-PRN with YOLOv3 head" or "Darknet53 with YOLOv3-PRN head"?

@WongKinYiu  Would you please share the cfg of "Darknet53 with YOLOv3-PRN head" to understand the difference? 

By the way, what is the main difference between the new CSPDarknet53 backbone and Darknet53-PRN backbone?

**Comment by WongKinYiu** [comment]
Hello, 

We only apply PRN on Darknet53 backbone, which is [`Darknet53-PRN with YOLOv3 head`](https://github.com/WongKinYiu/PartialResidualNetworks/issues/5#issuecomment-532937359).
We do not have cfg of `Darknet53 with YOLOv3-PRN head`. https://github.com/WongKinYiu/PartialResidualNetworks/issues/5#issuecomment-532002269

The main difference is that the channel number in each stage of Darknet53-PRN will linear decrease to a fraction.
![image](https://user-images.githubusercontent.com/12152972/73601134-d342c200-4596-11ea-945f-5a1936beed43.png)

**Comment by reactivetype** [comment]
Thanks @WongKinYiu  for the clarification. I actually referred to main conceptual differences between Darknet53-PRN and the new CSPNet version (CSDarknet53).

**Comment by WongKinYiu** [comment]
PRN aims to maximize the combinations of gradient of different layers.
CSPNet aims to maximize the combinations of different layers and minimize the mutual information of gradient of different layers.
